<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Form</class>
    <widget class="QWidget" name="Form">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>302</width>
                <height>204</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Form</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
            <property name="leftMargin">
                <number>2</number>
            </property>
            <property name="topMargin">
                <number>2</number>
            </property>
            <property name="rightMargin">
                <number>2</number>
            </property>
            <property name="bottomMargin">
                <number>2</number>
            </property>
            <item>
                <widget class="QGroupBox" name="config">
                    <property name="sizePolicy">
                        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                            <horstretch>0</horstretch>
                            <verstretch>0</verstretch>
                        </sizepolicy>
                    </property>
                    <property name="maximumSize">
                        <size>
                            <width>16777215</width>
                            <height>130</height>
                        </size>
                    </property>
                    <property name="title">
                        <string>Settings</string>
                    </property>
                    <layout class="QGridLayout" name="settings_area">
                        <item row="1" column="7">
                            <widget class="QPushButton" name="update_table">
                                <property name="sizePolicy">
                                    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                                        <horstretch>0</horstretch>
                                        <verstretch>0</verstretch>
                                    </sizepolicy>
                                </property>
                                <property name="text">
                                    <string/>
                                </property>
                            </widget>
                        </item>
                        <item row="1" column="6">
                            <layout class="QGridLayout" name="gridLayout">
                                <property name="sizeConstraint">
                                    <enum>QLayout::SetMinimumSize</enum>
                                </property>
                                <property name="spacing">
                                    <number>3</number>
                                </property>
                                <item row="1" column="2">
                                    <widget class="QSpinBox" name="symmetry">
                                        <property name="statusTip">
                                            <string>Number of subunits (type of symmetry)</string>
                                        </property>
                                        <property name="suffix">
                                            <string/>
                                        </property>
                                        <property name="minimum">
                                            <number>1</number>
                                        </property>
                                    </widget>
                                </item>
                                <item row="1" column="1">
                                    <widget class="QLabel" name="times">
                                        <property name="text">
                                            <string>x</string>
                                        </property>
                                        <property name="alignment">
                                            <set>Qt::AlignCenter</set>
                                        </property>
                                    </widget>
                                </item>
                                <item row="1" column="0">
                                    <widget class="QSpinBox" name="subunit_count">
                                        <property name="statusTip">
                                            <string>Number of domains per subunit</string>
                                        </property>
                                        <property name="suffix">
                                            <string/>
                                        </property>
                                        <property name="minimum">
                                            <number>2</number>
                                        </property>
                                        <property name="maximum">
                                            <number>999</number>
                                        </property>
                                    </widget>
                                </item>
                                <item row="1" column="4">
                                    <widget class="QSpinBox" name="total_count">
                                        <property name="enabled">
                                            <bool>false</bool>
                                        </property>
                                        <property name="statusTip">
                                            <string>Total number of domains</string>
                                        </property>
                                        <property name="readOnly">
                                            <bool>false</bool>
                                        </property>
                                        <property name="buttonSymbols">
                                            <enum>QAbstractSpinBox::NoButtons</enum>
                                        </property>
                                        <property name="suffix">
                                            <string/>
                                        </property>
                                        <property name="minimum">
                                            <number>0</number>
                                        </property>
                                        <property name="maximum">
                                            <number>999999</number>
                                        </property>
                                    </widget>
                                </item>
                                <item row="1" column="3">
                                    <widget class="QLabel" name="equals">
                                        <property name="text">
                                            <string>=</string>
                                        </property>
                                        <property name="alignment">
                                            <set>Qt::AlignCenter</set>
                                        </property>
                                    </widget>
                                </item>
                                <item row="0" column="0">
                                    <widget class="QLabel" name="label_2">
                                        <property name="text">
                                            <string>Domains</string>
                                        </property>
                                    </widget>
                                </item>
                                <item row="0" column="2">
                                    <widget class="QLabel" name="label">
                                        <property name="text">
                                            <string>Subunits</string>
                                        </property>
                                    </widget>
                                </item>
                                <item row="0" column="4">
                                    <widget class="QLabel" name="label_3">
                                        <property name="text">
                                            <string>Total</string>
                                        </property>
                                    </widget>
                                </item>
                                <item row="2" column="3" colspan="2">
                                    <widget class="QLabel" name="label_6">
                                        <property name="text">
                                            <string>Target M/R</string>
                                        </property>
                                    </widget>
                                </item>
                                <item row="4" column="3" colspan="2">
                                    <widget class="QDoubleSpinBox" name="target_M_over_R">
                                        <property name="enabled">
                                            <bool>true</bool>
                                        </property>
                                        <property name="statusTip">
                                            <string>To create a closed tube M/R must be this</string>
                                        </property>
                                        <property name="styleSheet">
                                            <string notr="true"/>
                                        </property>
                                        <property name="readOnly">
                                            <bool>true</bool>
                                        </property>
                                        <property name="buttonSymbols">
                                            <enum>QAbstractSpinBox::NoButtons</enum>
                                        </property>
                                        <property name="decimals">
                                            <number>3</number>
                                        </property>
                                        <property name="maximum">
                                            <double>999.000000000000000</double>
                                        </property>
                                    </widget>
                                </item>
                                <item row="2" column="2">
                                    <widget class="QLabel" name="label_5">
                                        <property name="text">
                                            <string>M/R</string>
                                        </property>
                                    </widget>
                                </item>
                                <item row="4" column="2">
                                    <widget class="QDoubleSpinBox" name="M_over_R">
                                        <property name="enabled">
                                            <bool>true</bool>
                                        </property>
                                        <property name="statusTip">
                                            <string>Sum of all ms from all subunits</string>
                                        </property>
                                        <property name="styleSheet">
                                            <string notr="true"/>
                                        </property>
                                        <property name="readOnly">
                                            <bool>true</bool>
                                        </property>
                                        <property name="buttonSymbols">
                                            <enum>QAbstractSpinBox::NoButtons</enum>
                                        </property>
                                        <property name="prefix">
                                            <string/>
                                        </property>
                                        <property name="decimals">
                                            <number>3</number>
                                        </property>
                                        <property name="minimum">
                                            <double>1.000000000000000</double>
                                        </property>
                                        <property name="maximum">
                                            <double>999.000000000000000</double>
                                        </property>
                                    </widget>
                                </item>
                                <item row="2" column="0">
                                    <widget class="QLabel" name="label_4">
                                        <property name="text">
                                            <string>M</string>
                                        </property>
                                    </widget>
                                </item>
                                <item row="4" column="0">
                                    <widget class="QSpinBox" name="M">
                                        <property name="enabled">
                                            <bool>false</bool>
                                        </property>
                                        <property name="statusTip">
                                            <string>Sum of all ms across all subunits</string>
                                        </property>
                                        <property name="styleSheet">
                                            <string notr="true">QSpinBox::disabled{background: rgb(255, 255, 255);
                                                color: rgb(0, 0, 0)}
                                            </string>
                                        </property>
                                        <property name="buttonSymbols">
                                            <enum>QAbstractSpinBox::NoButtons</enum>
                                        </property>
                                        <property name="minimum">
                                            <number>1</number>
                                        </property>
                                        <property name="maximum">
                                            <number>999</number>
                                        </property>
                                    </widget>
                                </item>
                            </layout>
                        </item>
                        <item row="1" column="0">
                            <spacer name="horizontalSpacer">
                                <property name="orientation">
                                    <enum>Qt::Horizontal</enum>
                                </property>
                                <property name="sizeHint" stdset="0">
                                    <size>
                                        <width>40</width>
                                        <height>20</height>
                                    </size>
                                </property>
                            </spacer>
                        </item>
                    </layout>
                </widget>
            </item>
            <item>
                <widget class="Line" name="line">
                    <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                    </property>
                </widget>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
